test-forms/lakeshorelearning_checkout.html

```diff
@@ -55,7 +55,7 @@
                             <input class="form-control" type="text" maxlength="80"
                                    automation-id="co_bill_cardholder_name"
                                    data-bind="textInput: payments.createPayment().cardHolderName, validateInput:'LtrCh1'"
-                                   data-manual-scoring="fullName">
+                                   data-manual-scoring="cardName">
```

Comment:
The fix highlighted it a wrong scoring (harmless, it's still the name 🤷‍♂️), so I changed it to the correct one.